r/R/compute.R

```diff
@@ -58,13 +58,12 @@ scalar_aggregate <- function(FUN, ..., na.rm = FALSE) {
     if (FUN %in% c("mean", "sum")) {
       # Arrow sum/mean function always drops NAs so handle that here
       # https://issues.apache.org/jira/browse/ARROW-9054
-      return(Scalar$create(NA_integer_, type = a$type))
+      return(Scalar$create(NA_real_))
     }
   }
 
-  if (inherits(a$type, "Boolean")) {
-    # Bool sum/mean not implemented so cast to int
-    # https://issues.apache.org/jira/browse/ARROW-9055
+  if (inherits(a$type, "Boolean") && FUN %in% "minmax") {
+    # Bool minmax not implemented so cast to int
```

Comment:
@nealrichardson just added support for minmax

Reply:
👍 I removed this special case altogether now, thanks.